sound/soc/intel/common/soc-acpi-intel-bxt-match.c

```diff
@@ -46,6 +46,11 @@ static struct snd_soc_acpi_codecs bxt_codecs = {
 	.codecs = {"MX98357A"}
 };
 
+#define drv_name common.drv_name
+#define fw_filename common.fw_filename
+#define sof_fw_filename common.sof_fw_filename
+#define sof_tplg_filename common.sof_tplg_filename
+
```

Comment:
@dbaluta whats the intention here? this looks wrong. If the intention is to have a common definition to you dont have to change the machine descriptors below, then it is incorrect and isnt going to work as intended.

Reply:
Yes, exactly the intention is to avoid changing of the machine descriptors. This looks like a little bit of abusing the macros usage but for RFC I did wanted my changes to be small. 

Actually, I did the correct changes myself for one file but given the amount of changes needed I got back and used macros.

And also, it works! It looks a little bit dangerous thoug. but it works! :)

Reply:
if you do not feel like it modifying all that by hand you could have the pre-processor do the bulk of it:
`make sound/soc/intel/common/soc-acpi-intel-byt-match.i`

Reply:
@groncarolo thanks, that's a smart idea :)